src/LeanCode.ContractsGenerator.Tests/ExampleBased/Notifications/NotificationTagGenerators.cs

```diff
@@ -0,0 +1,118 @@
+using Xunit;
+using CompilationTagGenerator = LeanCode.ContractsGenerator.Generation.NotificationTagGenerator;
+using ContractsTagGenerator = LeanCode.Contracts.NotificationTagGenerator;
+
+namespace LeanCode.ContractsGenerator.Tests.ExampleBased.Notifications;
+
+public class NotificationTagGenerators
+{
+    private const string Namespace = "LeanCode.ContractsGenerator.Tests.ExampleBased.Notifications";
+    private const string Class1FullName = $"{Namespace}.Class1";
```

Comment:
`nameof` is your friend here.

Reply:
Can't really use it for generic types, I exposed `GetSimpleName()` instead.

Reply:
You can - `nameof(Class1<object>)` will still return `Class1` :)